quantifier/src/main/java/edu/illinois/cs/cogcomp/quant/lbj/QuantitiesClassifier$$1.java

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
BTW, except NER, all the lbjava-based modules in the project have their lbjava-based classes checked in (e.g. POS).